src/coreclr/jit/target.h

```diff
@@ -146,13 +150,15 @@ enum _regNumber_enum : unsigned
     ACTUAL_REG_COUNT = REG_COUNT - 1 // everything but REG_STK (only real regs)
 };
 
-enum _regMask_enum : unsigned
+enum _regMask_enum : unsigned __int64
 {
-    RBM_NONE = 0,
+    RBM_NONE    = 0,
+    RBM_LOWSIMD = 1ULL << 63,
```

Comment:
I believe we have a macro `UI64(1)` that is preferred here.

It might also be nice if we used `uint64_t` for new code, rather than `unsigned __int64`, but there isn't any consistency there at the moment...